### ISSUE TITLE
Embedded config with custom artficat definitions broken

### ISSUE THREAD

**Issue by heck-gd** [Velocidex/velociraptor#376]
When packing a config such as the following into Velociraptor, running the repacked tool yields an error that the artifcat is unknown.
```yaml
autoexec:
  argv: ["artifacts", "collect", "-v", "AcquireAndUpload"]
  artifact_definitions:
    - name: AcquireAndUpload
      parameters:
      sources:
```
`error: Artifact AcquireAndUpload not known.`

This was working in 0.4.2, and it broke in commit 363b784 _or_ at some point afterwards (most commits from May panic when repacked with a config; this was only fixed very recently).

**Comment by scudette** [comment]
Yes thanks for reporting! I can confirm it is a regression bug in 0.4.3

**Comment by scudette** [comment]
This should be fixed in 0.4.4
